libs/m4i-data-management/README.md

```diff
@@ -0,0 +1,40 @@
+# M4I Data Management
+
+This library contains all core functionality around data management for Models4Insight.
+
+## Installation
```

Comment:
These instructions are no longer correct in the context of this repository. Let's update them or remove them.